server/src/providers/project/index.ts

```diff
@@ -40,34 +45,60 @@ export async function initialise() {
 }
 
 async function loadWorkspace() {
-	const uris = await getProjectFiles();
+	const workspaces = await connection.workspace.getWorkspaceFolders();
+
+	console.log(workspaces);
+
+	if (workspaces) {
+		let uris: string[] = [];
+
+		workspaces.forEach((workspaceUri => {
+			const folderPath = URI.parse(workspaceUri.uri).path;
+
+			console.log(`Starting search of: ${folderPath}`);
+			const files = glob.sync(projectFilesGlob, {
```

Comment:
Previously we were depending on the client extension to fetch a list of files. This was causing a major hangup. Now, we have the backend process do it.